test/distributed/_sharded_tensor/test_sharded_tensor.py

```diff
@@ -1904,10 +1886,6 @@ def test_init_from_local_shards_invalid_local_shards(self):
             placement=f"rank:{self.rank}/cuda:{self.rank}"
         )
 
-        empty_local_shards = []
-        with self.assertRaisesRegex(ValueError, 'have no local shards on all ranks'):
-            sharded_tensor = _sharded_tensor.init_from_local_shards(empty_local_shards, [10, 10], init_rrefs=True)
-
```

Comment:
Is this test no longer relevant?

Reply:
There were some issues when I try to run this test, mostly because if everywhere is none, we ended up having the `local_shards_device` be CPU, but we are calling `all_gather_object` on NCCL pg, let me see what I can do to make this legit.

Reply:
just updated with removing the `local_shards_device` and check with `isinstance` instead, I think this should make it cleaner :)